tests/wicked/before_test.pm

```diff
@@ -88,6 +88,13 @@ sub run {
             }
             assert_script_run('./autogen.sh ',       timeout => 600);
             assert_script_run('make ; make install', timeout => 600);
+        } elsif (my $wicked_repo = get_var('WICKED_REPO')) {
+            zypper_ar($wicked_repo, name => 'wicked_repo', no_gpg_check => 1);
+            zypper_call('in --force -y --repo wicked_repo --from wicked_repo --allow-vendor-change  --allow-downgrade  wicked', log => 1);
```

Comment:
```
-r, --repo alias|name|#|URI
               Work  only  with  the  repository  specified  by  the  alias, name, number or URI. This option can be used multiple times. + Using --repo is
               discouraged as it currently hides unmentioned repositories from the resolver, leading to inexpertly decisions. In  the  future  --repo  will
               become an alias for --from.
```

from `man zypper ` .  so at least according documentation `--from` only should be enough